app.js

```diff
@@ -72,69 +66,6 @@ const addAccountSid = (req, res, next) => {
 
 app.use('/v0/accounts/:accountSid', addAccountSid, apiV0);
```

Comment:
This middleware (`addAccountSid`) adds the `accountSid` being provided in the url to the request object. That way, the `accountSid` is allways carried as part of the url each helpline uses, and we can forget about sending it from each api call. Does this makes sense?